draft-ietf-rtcweb-jsep.xml

```diff
@@ -319,6 +319,19 @@ ipr="trust200902">
         answerer, who first calls setRemoteDescription(sdp [offer]) and
         then later setLocalDescription(sdp [answer]).</t>
 
+        <t>During the offer/answer exchange (i.e., in states other than
+        stable), the outstanding offer is "pending" at the offerer
```

Comment:
"stable" has not yet been defined at this point in the document.

Also suggest the phrasing 'is considered to be "pending"'